server/entity/MediaRequest.ts

```diff
@@ -538,7 +538,9 @@ export class MediaRequest {
         const tvdbId = series.external_ids.tvdb_id ?? media.tvdbId;
 
         if (!tvdbId) {
-          this.handleRemoveParentUpdate();
+          const requestRepository = getRepository(MediaRequest);
+          await mediaRepository.remove(media);
```

Comment:
By deleting the parent media, it should cascade and delete all related requests.

Reply:
This isn't the case. If we don't remove it from the reqest repository as well, it results in the media having a pending status whilst also not listing itself in requests. So some sort of failed state.

Reply:
How can the media have a pending state if it's deleted? There are foreign keys that tie the requests to the media so you can't delete the media without deleting its relations. That's what the on delete cascade should be doing?

Reply:
I don't know. But you can try it yourself.